tests/testthat/test-normalize_SE.R

```diff
@@ -50,4 +50,17 @@ test_that("normalize_SE works as expected", {
   se2 <- normalize_SE(se, "single-agent", nested_confounders = 
                         c(gDRutils::get_SE_identifiers(se, "barcode", simplify = TRUE), "masked"))
   expect_s4_class(se2, "SummarizedExperiment")
+  
+  
+  ctrl_df$Barcode <- NULL
```

Comment:
Please comment on these NULLs

Reply:
It's additional unit test added here: https://github.com/gdrplatform/gDRcore/pull/98#discussion_r1244989236